.circleci/config.yml

```diff
@@ -55,10 +55,39 @@ jobs:
           paths:
               - ~/.cargo
       - run: ./internal/ci/rust_test
+  sgx:
+    working_directory: /go/src/github.com/smartcontractkit/chainlink
+    docker:
+      - image: smartcontract/builder:1.0.12
+    environment:
+      SGX_ENABLED: yes
+      # XXX: These are set in the base image but circleci seems to wipe them out
+      PATH: "/root/.cargo/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/sgxsdk/bin:/opt/sgxsdk/bin/x64"
+      LD_LIBRARY_PATH: "/opt/sgxsdk/sdk_libs"
+      SGX_SDK: "/opt/sgxsdk"
+    steps:
+      - checkout
+      - restore_cache:
+          name: Restore Go Vendor Cache
+          key: v1-go-vendor-{{ checksum "Gopkg.lock" }}
+      - restore_cache:
+          key: cargo-cache
```

Comment:
Shouldn't we add a checksum to this key, perhaps of `Cargo.toml/lock`? If we ever change our rust dependencies, it looks like it'll be ignored.

Reply:
Ahh, good catch, I see there is a checksum for the Go Vendor cache I'll add one for the Cargo.lock